browser/src/GenePage/GenePageContainer.tsx

```diff
@@ -250,6 +256,53 @@ type Props = {
   geneIdOrSymbol: string
 }
 
+interface UnrolledVariantCooccurrenceCounts {
```

Comment:
I think you'll want to move this shaping logic elsewhere. Notice no other data is shaped in GenePageContainer. Looks like its main job is to fetch the data. 

To me it makes sense to do this shaping closer to where the data is used, like https://github.com/broadinstitute/gnomad-browser/blob/bfb9489fb10dfd065169090bc2d4c44c845b1ed8/browser/src/GenePage/VariantCooccurrenceCountsTable.tsx#L316

Reply:
@mattsolo1 I want to think about this a little more, but it seems like it might be more advantageous to reshape data closer to the query than the point of use, as a general rule. If I still think the same way on Monday I'll write a fuller explanation.

Reply:
@mattsolo1 I recommend that we start reshaping our data, when necessary, at the first opportunity once we have it--in other words, right after getting it back from the API. The essential reason is that that lets us keep the representation of that data, to the maximum degree possible, consistent throughout the frontend tier.

Reply:
@mattsolo1 are you cool with that? I can explain my reasoning in more detail if you want.

Reply:
That's fine. I'm not aware of too many instances when data must be shaped high in the component tree and then is used in several different places that would benefit from that shaping. I don't know how much it matters to have strict policies about this, I just thought the previous shaping logic looked out of place where it was in the gene page container. It may be a case-by-case basis, where sometimes it makes sense to shape right when we get it, and in other cases shape it in the component if the shape is strongly coupled to a particular view or something.